src/Websocket/Pusher.php

```diff
@@ -68,7 +68,7 @@ protected function __construct(
         bool $broadcast,
         bool $assigned,
         string $event,
-        $message = null,
+        string $message,
```

Comment:
looks it should be:

```php
?string $message
```

null is still the default option

Reply:
what? Put `?` In front of `string` ? Doesn't this report an error?

Reply:
OHHH, I found it!. This is a [new feature](https://www.php.net/manual/en/migration71.new-features.php) of PHP7.1. I had recommit a PR.